src/models/character.py

```
@@ -30,70 +95,426 @@ def get_total_attack(self) -> int:
+        attack: int,
+        defense: int,
+        exp_reward: int,
+        level: int,
+        art: str = None,
+    ):
+        super().__init__(name, description, health, attack, defense, level)
+        self.exp_reward = exp_reward
+        self.art = art
+        self.max_health = health
+
+    def get_drops(self) -> List["Item"]:  # type: ignore
```

Ruff (0.8.2)

Undefined name `Item`

(F821)